src/scripts/createJournalEntryHtml.js

```diff
@@ -0,0 +1,14 @@
+// Function that turns one entry object into an html string representation of that object
```

Comment:
By changing the arguments that this function accepts to a destructured object, we can remove the `journalEntry` object argument completely and just focus on what keys inside whatever object we pass to the function.

I also renamed `makeJournalEntryComponent` to `createJournalEntyHtml`, because as the old programming joke goes... 
> "There are only _two_ hard things in Computer Science: _naming things_, cache invalidation, and off-by-one errors" 🙄
But seriously, naming things can be hard, but since this function creates HTML rather than a component, i think that this is a valid/accurate change. Also, create is a common verb used for situations like this function, although arguably you could make the case (and I often do), that functions that return a thing should just be called that thing... sooo.... just `JournalEntryHtml`? Kind of like a React component? ... just more food for thought!